lnrpc/rpc.proto

```diff
@@ -537,6 +537,20 @@ message Utxo {
     int64 confirmations = 6;
 }
 
+message DestOutput {
```

Comment:
Same here, could just be `OutputDetail` and include the address (as a string). Then we can mark the `dest_addresses` as deprecated (but still keep the field for the moment to remain backward compatible).

Reply:
I have added the destination address to the `OutputDetail` structure. However, I do have a small reservation regarding how the address is calculated, see my other comment for the updated code.